contents/blog/product-market-fit-game.md

```diff
@@ -0,0 +1,351 @@
+
+If there's an obvious desire from the user to see if you can solve their problem, be ultra-accessible to these early potential users. People's emails are filled with junk, and they feel weirdly formal. Instead, add your customer to a WhatsApp group, or in a private Slack/Discord channel – whatever channel _they_ prefer. This communication will be critical throughout the rest of the levels.
+
+Try to move to level 3 as quickly as possible. Tim and I typically spent one week per idea in level 2. If you spend two weeks at level 2, it may be time to pivot back to level 1.
+
+#### Failure modes
+
+Work your way from top to bottom:
+
+* **You aren't working hard enough.** Realize no one else is going to do this for you. Set a goal for how many meetings you want to get. I got 2 meetings a day, 5 days a week while doing any other random tasks, like building a website. Try to meet in person if you can afford to do so – most communication is non-verbal, so it'll make it easier to learn what people are thinking. Move to a hub for your industry if you can. Also consider moving in with your co-founder, so you can focus 100% on the work for this intense period. Running out of momentum is your biggest risk.
+
+* **You aren't being clear what the problem is,** so it's hard to recruit users to validate it. Create a simple, single sentence description of the problem you think you're solving (ahead of doing the same for the _product_ you eventually build), and the user you are targeting. If what you're working on feels vague, you'll get a lukewarm response to your messages trying to book meetings. Do not use benefits-language here – users won't have the attention span to figure out what your product does.
+
+* **You are talking to the wrong users.** If so, no one will take calls, or if they do, they don't have your problem. Ask yourself – why did you have this problem, but those speaking to you don't? When you do get calls, ask who would have this problem. Change the user you are targeting.
+
+* **You aren't a good listener.** You're doing calls, users are seemingly confirming that they have your problem, but they don't use the product later. Why? Ask some open-ended questions to users in each call – perhaps you're close but not _quite_ right with your problem and target user statement. Ask your co-founder to shadow you on some calls, and give each other direct feedback afterwards on which questions worked well. This may help you find a new problem, in case...
+
+* **The problem you want to solve isn't there.** Pivot. Find a new problem to solve.
+
+### Level 3 - Validate your MVP by recruiting real users
+
+//TODO: Level 3 image here
+
+You should start Level 3 as soon as you have something for users to use. You need to _keep_ talking to users throughout this level, but now you can validate better because you have a product they can see.
```

Comment:
```suggestion
You should start level 3 as soon as you have something for users to use. You need to _keep_ talking to users throughout this level, but now you can validate better because you have a product they can see.
```